waspc/src/Wasp/AppSpec/Job.hs

```diff
@@ -49,13 +45,10 @@ data Schedule = Schedule
   }
   deriving (Show, Eq, Data)
 
-instance IsDecl Schedule
-
 -- These are optional executor-specific JSON options we pass
 -- directly through to the executor when submitting jobs.
 data ExecutorOptions = ExecutorOptions
   { pgBoss :: Maybe JSON,
-    simple :: Maybe JSON
```

Comment:
We don't use this as far as I can see, and it's not documented.

Reply:
Origin story:
<img width="679" alt="Screenshot 2024-09-16 at 15 09 21" src="https://github.com/user-attachments/assets/a22b54ac-299a-4868-a829-caaba7bb35ec">